xchange-bitmex/src/test/java/info/bitrich/xchangestream/bitmex/BitmexDeadManSwitchTest.java

```diff
@@ -62,13 +58,13 @@ public void testDeadmanSwitch() throws Exception {
 
         final BitmexStreamingMarketDataService streamingMarketDataService = (BitmexStreamingMarketDataService) exchange.getStreamingMarketDataService();
 //        streamingMarketDataService.authenticate();
-        CurrencyPair xbtUsd = CurrencyPair.XBT_USD;
+        CurrencyPair xbtUsd = exchange.determineActiveContract(CurrencyPair.XBT_USD.base.toString(), CurrencyPair.XBT_USD.counter.toString(), PERPETUAL);
```

Comment:
`determineActiveContract` takes "BTC" rather than "XBT" I think (or should do)...

Reply:
Ah no, works with either.  Cool.